firebase_admin/_auth_utils.py

```diff
@@ -243,7 +253,7 @@ def _parse_error_body(response):
         pass
 
     # Auth error response format: {"error": {"message": "AUTH_ERROR_CODE: Optional text"}}
-    code = error_dict.get('message')
+    code = error_dict.get('message') if isinstance(error_dict, dict) else None
```

Comment:
I could be wrong, but this doesn't seem very pythonic to me checking `isinstance`. Is it ever valid for `error_dict` to not be 'None' and also not support the `get` method?

Reply:
As long as the server response format doesn't change we should be fine. I'm being a little too defensive here. In general, it's not clear what the Auth response format would be in the event of a major outage or similar issue.